src/features/formRules/fetchRulesSagas.ts

```diff
@@ -33,7 +33,13 @@ export function* fetchRuleModelSaga(): SagaIterator {
 
     yield put(FormRulesActions.fetchFulfilled({ ruleModel: ruleModelFields }));
   } catch (error) {
-    yield put(FormRulesActions.fetchRejected({ error }));
-    yield put(QueueActions.dataTaskQueueError({ error }));
+    if (error.message?.includes('404')) {
+      yield put(FormRulesActions.fetchRejected({ error: null }));
+      window.logWarn('RuleHandler not found:\n', error);
```

Comment:
Again, info. This should not be a problem (hopefully soon, this should be the case for all new apps)